internal/sponge/item/item_test.go

```diff
@@ -159,3 +160,41 @@ func TestGetAll(t *testing.T) {
 		}
 	}
 }
+
+// TestInferId tests the inference of an item_id from type and source id.
```

Comment:
Wrong comment. golint should have caught this. Please validate you are running goling on save.